### ISSUE TITLE
GitX refers to nonexistent objc brush of SyntaxHighlighter, ext to brush mapping is broken

### ISSUE THREAD

**Issue by nanoant** [rowanj/gitx#127]
For `.m` files (probably it should also detect `.h`) GitX currently sets up `objc` brush identifier for _SyntaxHighlighter_ in `fileview.js`, however as far I can tell such brush does not exists.

So either it needs to be created or it should use `cpp` or `c` instead.

Right now all `.m` has absolutely no formatting.

![Zrzut ekranu 2013-01-16 o 18 30 10](https://f.cloud.github.com/assets/103067/72078/6a34cc1c-6002-11e2-9212-ecdb278ee13d.png)

Also `var suffix_map` hash is completely broken in `fileview.js`. Value for `h` key is set twice, first to `objc` then below to `cpp`, effect is that `h` files are highlighted by `cpp` brush.

**Comment by rowanj** [comment]
The cause of the title issue is that when I upgraded to upstream SyntaxHighlighter, I missed that the ObjC brush wasn't actually a part of that package; I'll put out a new build tonight with the brush added back in.

For this to work again, [shBrushObjC.js](https://raw.github.com/scottdensmore/ObjectiveCSyntaxHighlighter/master/scripts/shBrushObjC.js) just needs to go back into `GitX.app/Contents/Resources/html/lib/syntaxhighlighter/scripts/`.

I had evaluated the .h file issue previously, and I believe at the time decided that the C++ brush did a better job on Objective-C *.h files than the other way around; but this could just be that my opinion is weighted toward C++.  Obviously I missed cleaning up that double-key issue.
